cranelift-codegen/src/verifier/mod.rs

```diff
@@ -276,6 +289,47 @@ impl<'a> Verifier<'a> {
         }
     }
 
+    /// Build the context of an instruction for better errors.
+    #[inline]
+    fn context(&self, inst: Inst) -> Option<String> {
```

Comment:
Can the body of this function be inlined in `append_error`, which is its only one caller iiuc, instead of having a separate method for this?